src/parseAndModify/gistify.js

```diff
@@ -0,0 +1,28 @@
+const { post } = require('../http');
+const blockCodeRe = /```([a-z]*)\n([\s\S]*?)\n```/g;
+const blockCodePlaceholder = 'bH9IDSD1p7CK';
+const blockCodePlaceholderRe = new RegExp(blockCodePlaceholder, 'g');
+
+
+const createGist = async ({ codeString, _ext }) => {
+  const ext = _ext ? `.${_ext}` : '';
+  try {
+    const { id } = await post('https://api.github.com/gists', {
+      data: { files: { [`buildo${ext}`]: { content: codeString } } }
```

Comment:
I don't particularly love `buildo.scala` etc. I think `snippet.scala` or `gist.scala` would be better